### PR TITLE
Hidden label naked checkbox

### PR DESCRIPTION
**Pull request by ivarni**
This does not have a corresponding issue for reasons of personal lazyness but mainly, I would like to render (and position) a checkbox without any visible label without having to work around the padding/margin that the label would have had if it was there. Which it isn't. But the padding is. Which I wantn't.

### CODE REVIEW

File: packages/ffe-checkbox-react/src/Checkbox.js [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+        ...rest
+    } = props;
 
     const id = props.id || `checkbox-${hash()}`;
     const labelProps = {
         className: classNames({
             'ffe-checkbox': true,
             'ffe-checkbox--inline': inline,
             'ffe-checkbox--no-margin': noMargins,
+            'ffe-checkbox--hidden-label': hiddenLabel,
```
Comment (wkillerud) [inline packages/ffe-checkbox-react/src/Checkbox.js:23]: Is it too magic if this class and `hiddenLabel` value is derived from the fact that `aria-label !== undefined`?

Comment (ivarni) [reply packages/ffe-checkbox-react/src/Checkbox.js:23]: What if someone wants an image as a label?

Comment (ivarni) [reply packages/ffe-checkbox-react/src/Checkbox.js:23]: I suppose you could argue that `alt` on the image would work but I would personally still prefer this to not be magic.

Comment (selbekk) [reply packages/ffe-checkbox-react/src/Checkbox.js:23]: What about `--no-label`, to keep with the `--no-margin` modifier? Slightly different use case of course, but it's nice to have similar naming.

**Review by selbekk — DISMISSED** [review]
LGTM!

File: packages/ffe-checkbox-react/src/Checkbox.js (hunk 1 above)
Comment (ivarni) [reply packages/ffe-checkbox-react/src/Checkbox.js:23]: I can change it to that if that's what you prefer, but I didn't want to encourage anyone to have a checkbox with no label what-so-ever so I went with hidden and added the bit about `aria-label`.

Comment (wkillerud) [reply packages/ffe-checkbox-react/src/Checkbox.js:23]: The current name is fine by me - like Ivar says we don't want to encourage not using a label, even by name :)

File: packages/ffe-checkbox-react/src/Checkbox.js [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+import React, { Fragment } from 'react';
 import { bool, node, string, func, oneOfType } from 'prop-types';
 import { v4 as hash } from 'uuid';
 import classNames from 'classnames';
 
 export default function CheckBox(props) {
-    const { children, inline, invalid, label, noMargins, ...rest } = props;
+    const {
+        children,
+        hiddenLabel,
```
Comment (wkillerud) [inline packages/ffe-checkbox-react/src/Checkbox.js:9]: > `9:9  error  'hiddenLabel' is missing in props validation  react/prop-types`

**Review by wkillerud — DISMISSED** [review]
I'm happy once Travis is happy :)

File: packages/ffe-checkbox-react/src/Checkbox.js (hunk 1 above)
Comment (selbekk) [reply packages/ffe-checkbox-react/src/Checkbox.js:23]: Yeah I don't feel strongly either way, Go with `hidden`

File: packages/ffe-checkbox-react/src/Checkbox.js (hunk 2 above)
Comment (ivarni) [reply packages/ffe-checkbox-react/src/Checkbox.js:9]: We don't lint on precommit? 😨

Comment (ivarni) [reply packages/ffe-checkbox-react/src/Checkbox.js:9]: Fixed naow

**Review by selbekk — APPROVED** [review]

**Pull request merged**